lxc.md

```diff
@@ -123,7 +123,6 @@ Set up hostfiles on each box with hardcoded IP addresses
 
 
 ```
-127.0.0.1 localhost n1 n1.local
```

Comment:
I just followed the guide https://github.com/coreos/etcd/blob/master/Documentation/clustering.md#static where this doesn't seem to be an issue. I'll ping the etcd crowd and see if they can offer any advice.